src/actions/filter.ts

```diff
@@ -0,0 +1,26 @@
+import {FilterTransform} from 'vega-lite/build/src/transform';
+import {ReduxAction} from './redux-action';
+
+export type FilterAction = AddRangeFilter | AddOneOfFilter | RemoveRangeFilter | RemoveOneOfFilter ;
+
+export const ADD_RANGE_FILTER = 'ADD_RANGE_FILTER';
```

Comment:
For consistency, let's name this `FILTER_ADD_RANGE`.    (Use the part of state modified as prefix)